include/taskolib/Step.h

```diff
@@ -318,9 +341,10 @@ class Step
 
     /**
      * Execute the Lua script, throwing an exception if anything goes wrong.
-     * \see execute(Context&, CommChannel*, StepIndex)
+     * \see execute(Context&, CommChannel*, StepIndex, std::string&)
      */
-    bool execute_impl(Context& context, CommChannel* comm_channel, StepIndex index);
+    bool execute_impl(Context& context, CommChannel* comm_channel, StepIndex index,
+        std::string& step_setup);
```

Comment:
example of extra parameter... see above.

Reply:
Since we have the discussion about moving the setup script to `Context` the additional `std::string` will be possible removed. If it still is included in this member function I will add a comment.